slo_generator/report.py

```diff
@@ -256,7 +259,7 @@ def get_sli(self, data):
             good_count, bad_count = NO_DATA, NO_DATA
         return sli_measurement, good_count, bad_count
 
-    def to_json(self):
+    def to_json(self) -> Dict[str, Any]:
```

Comment:
`dict` is probably enough here. Then you can remove the `Dict` and `Any` imports on line 21.